src/org/traccar/helper/DataConverter.java

```diff
@@ -26,7 +26,7 @@ private DataConverter() {
 
     public static byte[] parseHex(String string) {
         try {
-            return Hex.decodeHex(string);
+            return Hex.decodeHex(string.toCharArray());
```

Comment:
Why do we need this change?

Reply:
I had a build error on this line. It could be because I am using maybe an older version of java?
I am running JDK 1.8.0_161 on my computer. The toCharArray() function fixed the build error for me.

Reply:
This is a part of third party library and has nothing to do with version of Java that you are using. If it doesn't work, it means there is some issue with your SDK or development tools. This needs to be reverted.